README.rst

```diff
@@ -0,0 +1,223 @@
+======================
+Cookiecutter PyPackage
```

Comment:
Change the title : cookiecutter doesn't matter. May be some day we will change this engine for another ? Focus on the target - having an opinionated full python library project.

Reply:
Yes but on the other hand, the project name is `cookiecutter-pypackage`. Do you suggest to change the whole thing? Can this wait for future releases?

Reply:
Following exchange, will replace with Python library project template